### ISSUE TITLE
can not use json/math module

### ISSUE THREAD

**Issue by guochenglai** [pybee/voc#781]
**when i use math/json module i encountered the follow problems:**
# math module problems
### origine python code 
```
# coding=utf-8
import math

def absParam(x):
    return math.fabs(x) 
```

### The Code I called
```
 public static void testSimpleMath() {
        try {
            Int param = new Int(-1);
            SimpleMath.main(new String[1]);
            Object result = SimpleMath.absParam(param);
            System.out.println("result : " + result);
        } catch (Exception e) {
            System.out.println("testSimpleMath ");
            e.printStackTrace();
        }
    }
```
### Got the following exception
```
testSimpleMath 
NameError: name 'fabs' is not defined
	at org.python.java.Module.__getattribute__(Module.java:33)
	at python.SimpleMath.absParam(SimpleMath.py:5)
	at com.aliyun.yundun.demo.core.py.Function2Java.testSimpleMath(Function2Java.java:25)
	at com.aliyun.yundun.demo.core.py.Function2Java.main(Function2Java.java:15)
process one person cause exception name 'loads' is not defined

Process finished with exit code 0
```

# json module problems
### origine python code 
```
# coding=utf-8
import json

def parseName(person):
    try:
        jsonObject = json.loads(person)
        if "name" in person:
            return jsonObject["name"]
        else:
            return "not found"
    except BaseException as e:
        print("process one person cause exception %s" % e)
    return "no name found"
```

### The Code I called
```
try {
            Str json = new Str("");

            SimpleJson.main(new String[2]);

            SimpleJson.parseName(json);
        } catch (Exception e) {
            System.out.println("testSimpleJson ");
            e.printStackTrace();
        }
```

### Got the following exception
```
process one person cause exception name 'loads' is not defined

Process finished with exit code 0
```

**can anyone help me to resolve my problems !!!!**

**Comment by freakboy3742** [comment]
Unfortunately, VOC is currently only an implementation of the Python *language* - it doesn't include most of the Python standard library. 

The list of modules that are explicitly supported can be found [here](https://github.com/pybee/voc/tree/master/python/common/python). It *should* also be possible to run any module whose code is 100% python (i.e., no C module is involved - both `json` and `math` have C components), but many of the modules in the standard library [are known to have problems](https://github.com/pybee/voc/blob/master/tools/compile_stdlib.py#L14) due to limitations in the VOC compiler.

In short - VOC isn't "ready for production use" - it's in the early stages of development. We welcome any pull requests, but you want to use it right now, you're going to need to do a lot of debugging on your own.

**Comment by guochenglai** [comment]
Are there any other product like `voc` that can be used directly in production environment ?

**Comment by guochenglai** [comment]
@uranusjr  I have a calculation engine written in Java, but need to efficiently support the user to write Python scripts。I can not directly call python script in my java code。  This will start a process to execute this script, which is inefficient 。 so i need translate python code to java code 。

**Comment by eliasdorneles** [comment]
@guochenglai you should probably check out Jython, if you haven't already.
